src/index.html

```diff
@@ -11,8 +11,55 @@
       rel="stylesheet"
       href="./styles/index.scss"
```

Comment:
The href attribute should link to a `.css` file, not a `.scss` file. SCSS files are preprocessed and should be compiled into CSS files that the browser can understand.